lib/action_mailer/logged_smtp_delivery.rb

```diff
@@ -2,6 +2,8 @@
 require 'mail/network/delivery_methods/smtp'
 
 class ActionMailer::LoggedSMTPDelivery < Mail::SMTP
+  ULID_PATTERN = /[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}/.freeze
```

Comment:
```suggestion
  ULID_PATTERN = /[A-Z0-9]{26}/.freeze
```
^^ would this work?

Reply:
We had this `/[A-Z0-9]{26}/` regex initially but according to the spec - [ULID](https://github.com/ulid/spec#encoding ) value `excludes the letters I, L, O, and U to avoid confusion and abuse.` Hence we updated the regex it to `/[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}/` 🙏

Reply:
```suggestion
ULID_PATTERN = /[A-HJ-KMNV-Z0-9]{26}/.freeze
```
Would this be better? Also it would be to put the comment that we are omitting letters and why we are doing it

Reply:
Sure, I will add the comment about omitting the letters

Reply:
How about this regex? /[A-HJKMNV-Z0-9]{26}/.freeze